pyproject.toml

```diff
@@ -16,7 +16,7 @@ maintainers = [ "Tom Bocklisch <tom@rasa.com>",]
 homepage = "https://rasa.com"
 repository = "https://github.com/rasahq/rasa"
 documentation = "https://rasa.com/docs"
-classifiers = [ "Development Status :: 4 - Beta", "Intended Audience :: Developers", "License :: OSI Approved :: Apache Software License", "Programming Language :: Python", "Programming Language :: Python :: 3.6", "Programming Language :: Python :: 3.7", "Programming Language :: Python :: 3.8", "Topic :: Software Development :: Libraries",]
```

Comment:
the python classifiers are auto generated by [poetry](https://python-poetry.org/docs/pyproject/#classifiers):
> Note that Python classifiers are still automatically added for you and are determined by your python requirement.

Reply:
couldn't find a way to test this locally though, there is no debug command in poetry to do that 🤔

Reply:
so it will be added in again after a `poetry update`?

Reply:
poetry doesn't update the `pyproject.toml`, my understanding is that when we build a new version of rasa to publish on pypi, poetry reads the [python requirement here](https://github.com/RasaHQ/rasa/blob/4fd90e7b55a6e354a7dfd16286e5fe7f11b53f52/pyproject.toml#L66) and updates the classifiers on the fly, so that they end up in pypi

Reply:
aha ok - makes sense